1주차/14659/14659_java_포키.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int count = scanner.nextInt();
+        scanner.nextLine();
+        List<Integer> peeks = Arrays.stream(scanner.nextLine().split(" "))
+                .map(Integer::parseInt)
+                .collect(Collectors.toList());
+
+        int maxKill = 0;
+        for (int i = 0; i < count; i++) {
+            int kill = 0;
+            for (int j = i + 1; j < count; j++) {
+                if (peeks.get(i) < peeks.get(j)) {
+                    break;
+                }
+                kill++;
+            }
+            if (maxKill < kill) {
+                maxKill = kill;
+            }
```

Comment:
`Math`에서 제공하는 메서드를 활용하여 다음과 같이 풀 수도 있을 것 같아요 😁

```suggestion
            maxKill = Math.max(maxKill, kill);
```